package.json

```diff
@@ -25,6 +25,10 @@
     "oracle-e2e": "./oracle-e2e/run-tests.sh",
     "ui-e2e": "./ui-e2e/run-tests.sh",
     "clean": "rm -rf ./node_modules ./**/node_modules ./**/**/node_modules ./**/build",
+    "compile:contracts": "yarn workspace poa-parity-bridge-contracts run compile",
```

Comment:
Does it make sense to use `yarn run compile:contracts` in the lines 21 and 23?

Reply:
Good spot. Done.